### PR TITLE
[Oy2-8944] Create Measure 11: CCP-CH

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create CCP-CH using common question components.

#### Linked Issues to Close

[Oy2-8944](https://qmacbis.atlassian.net/browse/OY2-8944?atlOrigin=eyJpIjoiNjk3MDE4MWE5NDZkNDUwOGJjYzljNmZlZGIwNjM0MWYiLCJwIjoiaiJ9)

## Approach

Used common question components and established build patterns to create this measure.

#### Pull Request Creator Checklist

- [X] This PR has an associated issue or issues.
- [X] The associated issue(s) are linked above.
- [X] This PR meets all acceptance criteria for those issues.
- [X] This PR and linked issue(s) are adequately documented
- [X] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [X] Someone has been assigned this PR.
- [X] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Comment by abelzer212** [comment]
Screen Shot 2022-04-04 at 10 00 01 AM
Screen Shot 2022-04-04 at 9 59 48 AM
Screen Shot 2022-04-04 at 9 59 35 AM

Missing a few Validations:
1. The Three day Postpartum Rate must be smaller than the 60 day Postpartum rate within EACH category
2. The Three Days Postpartum Rate for Most Effective or moderately effective must be BIGGER than the Three Days postpartum Rate for Long-acting reversible Method of Contraception
3. The same as above but for Sixty days
4. All 3 of the above must also be applied in OMS

File: services/ui-src/src/measures/2021/CCPCH/validation.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      didCalculationsDeviate
+    ),
+    ...GV.validateNoNonZeroNumOrDenom(performanceMeasureArray, OPM, ageGroups),
+    ...GV.validateNumeratorsLessThanDenominators(
+      performanceMeasureArray,
+      OPM,
+      ageGroups
+    ),
+    ...GV.validateRequiredRadioButtonForCombinedRates(data),
+
```
Comment (abelzer212) [inline services/ui-src/src/measures/2021/CCPCH/validation.ts:44]: Here for the performance measure

**Review by abelzer212 — CHANGES_REQUESTED** [review]

File: services/ui-src/src/measures/2021/CCPCH/validation.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        OMSData(true),
+        PMD.qualifiers,
+        PMD.categories
+      ),
+      validationCallbacks: [
+        GV.validateDenominatorGreaterThanNumerator,
+        GV.validateDenominatorsAreTheSame,
+        GV.validateOneRateLessThanOther,
+        GV.validateRateNotZero,
+        GV.validateRateZero,
```
[the remark this answers is not in the document]
Comment (jphechter) [reply services/ui-src/src/measures/2021/CCPCH/validation.ts:64]: Included.

File: services/ui-src/src/measures/2021/CCPCH/validation.ts (hunk 1 above)
Comment (jphechter) [reply services/ui-src/src/measures/2021/CCPCH/validation.ts:46]: Included.

**Review by charmcitygavin — APPROVED** [review]

**Review by abelzer212 — DISMISSED** [review]

**Pull request merged**